### PR TITLE
#4569 Upgrade kafka from 2.4.1 to 2.5.0

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4569)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by jackyoh** [comment]
retry

**Comment by jackyoh** [comment]
retry

File: docker/build.yml [hunk 1]
```diff
@@ -78,22 +78,22 @@ services:
       context: .
       dockerfile: zookeeper.dockerfile
       args:
-        ZOOKEEPER_VERSION: 3.4.13
+        ZOOKEEPER_VERSION: 3.5.7
```
Comment (jackyoh) [inline docker/build.yml:81]: 在 apache repository 的 zookeeper 檔案名稱有修改了，很像多了 apache- 的 prefix，之前的版本沒有這個 prefix
https://archive.apache.org/dist/zookeeper/zookeeper-3.5.7/apache-zookeeper-3.5.7.tar.gz

https://archive.apache.org/dist/zookeeper/zookeeper-3.4.9/zookeeper-3.4.9.tar.gz

因此在 build image 時有可能失敗

Comment (vitojeng) [reply docker/build.yml:81]: @jackyoh Nice finding!

Comment (chia7712) [reply docker/build.yml:81]: 這個錯誤跟build很慢沒有關係，因為找不到libs的錯誤很快就會觸發（例如此次job才花了30分鐘）

Comment (jackyoh) [reply docker/build.yml:81]: 我還在找其它地方 (QA) 是否有問題。

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry collieIT

**Comment by jackyoh** [comment]
retry collieIT

**Comment by chia7712** [comment]
retry collieIT

**Comment by chia7712** [comment]
the flaky passes on my local so I'm going to merge it :)

**Pull request merged**